fluentd/templates/nginx_proxy.conf

```diff
@@ -0,0 +1,74 @@
+server {
+    listen 80;
+    listen [::]:80;
+    {% if config.get('server_name') -%}
+    server_name {{ config.server_name }};
+    {%- endif %}
+
+    # Redirect all HTTP requests to HTTPS with a 301 Moved Permanently response.
+    return 301 https://$host$request_uri;
+}
+
+server {
+    listen 443 ssl;
+    listen [::]:443 ssl;
+    {% if config.get('server_name') -%}
+    server_name {{ config.server_name }};
+    {%- endif %}
+
+    # certs sent to the client in SERVER HELLO are concatenated in ssl_certificate
+    ssl_certificate {{ ssl_directory }}/{{ config.cert_file }};
+    ssl_certificate_key {{ ssl_directory }}/{{ config.key_file }};
+    ssl_session_timeout 1d;
+    ssl_session_cache shared:SSL:50m;
+    ssl_session_tickets off;
+
+    # Diffie-Hellman parameter for DHE ciphersuites, recommended 2048 bits
+    ssl_dhparam {{ ssl_directory }}/dhparam.pem;
+
+    # modern configuration. tweak to your needs.
+    ssl_protocols TLSv1.2;
+    ssl_ciphers 'ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256';
+    ssl_prefer_server_ciphers on;
+
+    # HSTS (ngx_http_headers_module is required) (15768000 seconds = 6 months)
+    add_header Strict-Transport-Security max-age=15768000;
+
+    # OCSP Stapling ---
+    # fetch OCSP records from URL in ssl_certificate and cache them
+    ssl_stapling on;
+    ssl_stapling_verify on;
+
+    resolver 8.8.8.8;
+
+    {% if config.get('root_path') %}
+    root {{ config.root_path }};
+    {% endif %}
+    index index.html;
+
+    # start_extra_configs
+    # end_extra_configs
```

Comment:
Yes, these are left over from copy/pasting this out of the Kibana formula. I think I'll probably leave them in there though in case we want to make use of them in the future. Good catch :)